src/quo2/components/wallet/network_routing/style.cljs

```diff
@@ -0,0 +1,64 @@
+(ns quo2.components.wallet.network-routing.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.reanimated :as reanimated]))
+
+(defn container
+  [container-style theme]
+  (assoc container-style
+         :flex-direction   :row
+         :height           64
+         :background-color (colors/theme-colors colors/neutral-100-opa-5 colors/neutral-90 theme)
+         :border-radius    20
+         :overflow         :hidden))
+
+(defn max-limit-bar
+  [{:keys [opacity-shared-value color width]}]
+  (reanimated/apply-animations-to-style
+   {:opacity opacity-shared-value}
+   {:position           :absolute
+    :top                0
+    :bottom             0
+    :left               0
+    :background-color   (str color "10")
```

Comment:
Can you elaborate on this? If we need to set a color with opacity we have some functions in `colors` namespace (also 10 in hex would be like around 6% of opacity, is that the desired value?).

Reply:
oh yes! good catch!, I'll use `colors/alpha` :+1: